packages/squiggle-lang/scripts/run.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+const s = require("@quri/squiggle-lang");
+
+const p = s.SqProject.create();
```

Comment:
I'm not a big fan of one-character variable names

Reply:
Yeah. Fixed.